test_memory_diff/Matecat_New.tmx

```diff
@@ -6,10 +6,10 @@
       <prop type="matecat-tee">41698</prop>
       <prop type="matecat-match">MT</prop>
       <tuv xml:lang="en-GB">
-        <seg>Important legal notice, really important</seg>
+	  <seg>Importante avviso legale, veramente importante</seg>

```

Comment:
remove the tmx files from github and add them to the .gitignore, both old, new and diff file